files/events.yaml

```diff
@@ -1,4 +1,37 @@
 ---
+- title: "Richard Stallman - Copyright vs Community in the Age of Computer Networks"
+  date: 2014-11-07
+  time: '18:00'
+  speaker: Dr. Richard Stallman
+  description: |
+      "Location: Portland State University Hoffman Hall, 1833 SW 11th Avenue, Portland, Oregon
+
+      Copyright developed in the age of the printing press, and was designed
+      to fit with the system of centralized copying imposed by the printing
+      press.  But the copyright system does not fit well with computer
+      networks, and only draconian punishments can enforce it.
+
+      The global corporations that profit from copyright are lobbying
+      for draconian punishments, and to increase their copyright powers,
+      while suppressing public access to technology.  But if we
+      seriously hope to serve the only legitimate purpose of
+      copyright--to promote progress, for the benefit of the
+      public--then we must make changes in the other direction.
+
+     Speaker Bio:
```

Comment:
Well, I had made a pull request on @finchd's repository:
https://github.com/finchd/acm.pdx.edu/pull/1
... so that if he accepts my pull request, this will be valid.

Since it's sat there over a month, I'll just do it myself now.